pom.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+       Copyright 2016-2017 the original author or authors.
```

Comment:
Should be `2017`. (not `2016 - 2017`)
@harawata Let's fix this by using maven-licence-plugin!